quantum/quantum.c

```diff
@@ -257,9 +257,6 @@ bool process_record_quantum(keyrecord_t *record) {
 #ifdef LEADER_ENABLE
             process_leader(keycode, record) &&
 #endif
-#ifdef COMBO_ENABLE
-            process_combo(keycode, record) &&
-#endif
```

Comment:
I'm not a fan of moving this handling to tmk_core. We're kind of working towards moving all keycode handling to the `process_keycode/` directory.

Reply:
Ah, I had one idea for this actually.

There could some pre tapping hook there that calls a function in quantum, like it is done in `process_record`. Then all the pre tapping quantum code would stay on the right side.

Reply:
Right, just implemented that.